_dev/front/front.scss

```diff
@@ -59,8 +59,17 @@
     width: 35px;
 }
 
+
+.blockreassurance_product >div {    
```

Comment:
```suggestion
.blockreassurance_product > div {    
```

Reply:
A space make CSS code easier to read. The problem now is `<div class="clearfix"></div>` in [TPL line 40](https://github.com/PrestaShop/blockreassurance/blob/dev/views/templates/hook/displayBlockProduct.tpl#L40) is affected by this CSS line, and create redundant space with Product Extra Info block below. 
I think it will be better if we add a classname in [TPL line 22](https://github.com/PrestaShop/blockreassurance/blob/dev/views/templates/hook/displayBlockProduct.tpl#L22)

Reply:
This is not related to the space right?

Reply:
Yes, it is not related to the space.
We'd better remove this line to avoid side effect and extra work, maybe we need it in the future improvement but not for issue 2188.